core/src/main/java/org/elasticsearch/search/sort/ScriptSortBuilder.java

```diff
@@ -189,7 +112,7 @@ public XContentBuilder toXContent(XContentBuilder builder, Params builderParams)
             builder.field("nested_path", nestedPath);
         }
         if (nestedFilter != null) {
-            builder.field("nested_filter", nestedFilter, params);
```

Comment:
Thanks for checking so closely, seems like this is a bug that was introduced earlier here: https://github.com/elastic/elasticsearch/commit/35a58d874ef56be50a0ad1d7bfb13edb4204d0a3

Switched to what is now called builderParams.